src/workerd/api/tests/global-scope-test.js

```diff
@@ -744,3 +746,12 @@ export const toStringTag = {
     strictEqual(new Headers()[internalFlag], internalFlag);
   },
 };
+
+export const validateGlobalThis = {
+  test() {
+    doesNotThrow(
```

Comment:
Using `doesNotThrow(...)` is a bit unnecessary. Just calling `util.inspect(globalThis)` should be fine here as if the inspection throws the test will fail.

Reply:
Removed `doesNotThrow`